### DISCUSSION TITLE
Back button when viewing track and albums should be at the 'start' (left for Left-To-Right languages)

### DISCUSSION THREAD

**Discussion by JeffersonBledsoe** [music-assistant/support#1646] [Feature requests and ideas]
### What version of Music Assistant has the issue?

2023.6.0b6

### The problem

The back button is currently at the top right. Personally this is quite unexpected as most apps use the top left and makes the app a bit more confusing. In CSS terms, we should be positioning it using `inline-start`, so that when using a RTL language, it is positioned on the top right.
<img width="2032" alt="Screenshot 2023-11-21 at 12 03 18 pm" src="https://github.com/music-assistant/hass-music-assistant/assets/30210785/eac0d26e-be6e-47ea-a6c4-fcd432217429">

### How to reproduce

1. Open Music Assistant
2. Open a track or album
3. Try to go 'back' to where you were previously

### What version of Home Assistant Core are your running

2023.11.x

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by OzGav** [comment]
This isn’t an issue. Moved to feature requests.

**Reply by JeffersonBledsoe** [reply]
Thanks! Wasn't sure if "UI/ UX" improvements were 'bugs' or 'features.
